nephio-ansible-install/README.md

```diff
@@ -38,11 +38,13 @@ all:
     cloud_user: <username that is used to access the VM>
     github_username: <GitHub username>
     github_token: <GitHub personal access token>
-    github_organization: <optional, if a GitHub organization is used this should be filled out with the GitHub organization>
+    github_uri: github.com <replace this if you have a different github domain name in case of organization hosted github>
+    github_organization: <if a GitHub organization is used this should be filled out with the GitHub organization, else provide your github username>
```

Comment:
Which file is consuming the `githhub_organization`?

Reply:
The previous role `github_organisation` is not changed. As we only have introduced `github_uri` so to include other github domains as well, mainly in the case of organization specific git hub domains:
e.g. https://github.sec.<vendor>.net/

Reply:
Thinking of this further, supporting different github domain name is out of scope for this PR. 
Will take it seperately in full. I have reverted it related changed.